configs/smart-commute-mntmo.nrel-op.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://smart-commute-mntmo-openpath.nrel.gov/api/",
```

Comment:
@shankari can you please confirm which `connectUrl` we want for Smart Commute? Thanks!

Reply:
The base URL is https://smart-commute-ebike-openpath.nrel.gov
Note that you also need to rename the config appropriately

Reply:
Resolved in https://github.com/e-mission/nrel-openpath-deploy-configs/pull/15/commits/fbe2cc78dbacd81d540201b1655299128200c0b7